src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -4031,7 +4016,7 @@ namespace WaterToAirHeatPumpSimple {
             }
 
         } else if (wahp.WAHPType == WatertoAirHP::Heating) {
-            if (wahp.RatedEntAirDrybulbTemp != DataSizing::AutoSize) {
+            if (wahp.RatedEntAirDrybulbTemp != DataSizing::AutoSize && wahp.HeatCapCurve != nullptr && wahp.HeatPowCurve != nullptr) {
```

Comment:
Issue correction is here on these 3 lines above in function CheckSimpleWAHPRatedCurvesOutputs

Reply:
Got it.  The function will be fine skipping this part, because it doesn't need to check the curves that don't exist.